lib/translate.tsx

```diff
@@ -153,6 +156,30 @@ function injectComponent(result, type) {
           )
           break
         }
+        case ComponentType.OrderedList: {
+          // Match all the <li> elements inside the <ol>
+          const liMatches = match[2].match(/<li(.*?)>(.*?)<\/li>/gim)
+          const items = liMatches
+            ? liMatches.map((liHtml, itemIndex) => {
+                const liLabel = getFirstMatch(liHtml, labelRegex)
+                const liClassName = getFirstMatch(liHtml, classNameRegex)
+                return (
+                  <li key={`item-${itemIndex}`} className={liClassName}>
+                    {liLabel}
+                  </li>
+                )
+              })
+            : []
```

Comment:
I think the `<span>` element is recognized and rendered before everything inside the `<li>` tag is rendered so thats why removing the `<span>` element allows it to work.